db/test/unit/src/org/netbeans/modules/db/explorer/DbDriverManagerTest.java

```diff
@@ -221,7 +221,7 @@ private static JDBCDriver createJDBCDriver() {
     
     private static JDBCDriver createDummyJDBCDriver(File dataDir) throws MalformedURLException {
         URL url = dataDir.toURL();
-        return JDBCDriver.create("test_driver", "DbDriverManagerTest DummyDriver", "DummyDriver", new URL[] { url });
+        return JDBCDriver.create("test_driver", "DbDriverManagerTest DummyDriver", "org.netbeans.modules.db.explorer.DbDriverManagerTest$DriverImpl", new URL[] { url });
```

Comment:
So what is the significance of this change? (If it is unrelated to "license" review, then I'd suggest to send it separately, for clarity.)

Reply:
It is related, as I run unittests and they failed. This was necessary to let them cleanly again. This is related to the change in the binaries-list, as the binaries are used by the unittests.

Reply:
So, the test was passing before other changes in the pull request and started to fail after the changes have been made?

Reply:
No - master fails this test also.

Reply:
If I were doing this, I'd probably do two commits - one license-related cleanup, and one test fix/cleanup. OTOH, I don't think this is important enough to block the change on it, so I'll leave the decision on you on this matter.

Reply:
Same here. I would prefer a split - first ee fix sll the thing eith regards to licensing etc. stuff and afterwards we start fixing failing tests (I assume we will find more)